lib/algora/accounts.ex

```diff
@@ -5,6 +5,8 @@ defmodule Algora.Accounts do
   alias Algora.{Repo, Restream}
   alias Algora.Accounts.{User, Identity, Destination, Entity}
 
+  @stream_key_expiration_days 30
```

Comment:
Why are we expiring stream keys? They should be valid forever unless the user resets